### ISSUE TITLE
Missing newActiveInParallel(String classname, Class<?>[] genericParameters, Object[][] constructorParameters, Node[] node, Active[] activity, MetaObjectFactory[] factory)

### ISSUE THREAD

**Issue by activeeon-bot** [ow2-proactive/programming#569]
<a href="https://jira.activeeon.com/browse/PROACTIVE-688" title="PROACTIVE-688">Original issue</a> created by <a href="mailto:fabrice.fontenoy_AT_sophia.inria.fr">Fabrice Fontenoy</a> on 05, Jun 2009 at 11:37 AM - PROACTIVE-688

<hr />

Currently, there is no newActiveInParallel method in PAAcitveObject with which we can create active objects in parallel using an activity object (implementing some of the three activity methods) or using a MetaObjectFactory object.

**Comment by activeeon-bot** [comment]
<a href="https://jira.activeeon.com/browse/PROACTIVE-688?focusedCommentId=18527">Original comment</a> posted by <a href="mailto:youri.bonnaffe_AT_activeeon.com">Youri Bonnaffe</a> on 12, Dec 2014 at 12:54 PM

<hr />

Marking old issues as won't fix, either because we don't support it anymore, or it is needed or it has been there for too long.

Feel free to reopen if needed.
